### DISCUSSION TITLE
make error in autotiler setup

### DISCUSSION THREAD

**Discussion by ArtemRedchych** [bitcraze/discussions#628] [Q&A] [answered]
Hello, can someone please help me with this problem? I am trying to set up [gap8 development environment](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#setup-development-environment)

Do you agree with the license (y/n) ? y
--2023-03-22 16:23:43--  https://pigsffcp.page.link/qbvQ/libtile.4.2.0.a
Resolving pigsffcp.page.link (pigsffcp.page.link)... 142.251.36.97, 2a00:1450:4014:80b::2001
Connecting to pigsffcp.page.link (pigsffcp.page.link)|142.251.36.97|:443... connected.
HTTP request sent, awaiting response... 400 Bad Request
2023-03-22 16:23:44 ERROR 400: Bad Request.

make[2]: *** [Makefile:23: Autotiler/LibTile.a] Error 1
make[2]: Leaving directory '/gap_sdk/tools/autotiler_v3'
make[1]: *** [gap8/gap8.mk:258: autotiler] Error 2
make[1]: Leaving directory '/gap_sdk'
make: *** [Makefile:42: all] Error 2

**Comment by knmcguire** [comment]
This seems to be a server issue on Greenwaves' side. Somebody reported a similar issue a while ago: https://github.com/bitcraze/aideck-gap8-examples/issues/117. But that had a different error message. 

@Yaooooo could you help out?

**Reply by ArtemRedchych** [reply]
I copied wrong link in licence agreement step. I did it again with correct link and everything work just fine

**Reply by knmcguire** [reply]
Thanks for letting us know. Could you copy this answer and propose it as an answer (and select it as answer) here underneath? Then this discussion is considered answered and it will be shown directly in the preview on the main page

**Comment by ArtemRedchych** [comment, marked as answer]
I copied wrong link in licence agreement step. I did it again with correct link and everything work just fine

**Reply by knmcguire** [reply]
thanks! now t[he leaderboard on the first page](https://github.com/orgs/bitcraze/discussions) has given you points for providing an answer to this question ;)
